src/components/drop-down/drop-down.tsx

```diff
@@ -70,15 +62,62 @@ export class DropDown extends React.Component<DropDownProps, {}> {
         });
 
         return (
-            <div data-automation-id="DROP_DOWN" {...rootProps}>
-                <DropDownInput selectedItem={this.props.selectedItem} onClick={this.props.onInputClick} />
-                <DropDownList
-                    selectedItem={this.props.selectedItem}
-                    items={this.props.items}
-                    open={!!this.props.open}
-                    onItemClick={this.onItemClick}
-                />
+            <div
+                {...rootProps}
+                onKeyDown={this.onKeyDown}
+                onFocus={this.onFocus}
+                tabIndex={this.props.tabIndex}
+                ref={dropdown => {
```

Comment:
this format always keeps ref to the first dom node you had. there is no need for the if condition